services/frontend-v3/src/components/experience/experienceItem/ExperienceItemView.jsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { injectIntl } from "react-intl";
+import { Row, Col, Divider, Avatar, List } from "antd";
+import { ContainerOutlined, DownOutlined, UpOutlined } from "@ant-design/icons";
+
+function ExperienceItem(props){
+    const styles = {
+        avatar: {
+          backgroundColor: "#007471",
+        },
+        experienceDescription: {
+            color: "rgba(0, 0, 0, 0.85)",
+            maxWidth:"525px"
```

Comment:
is there a reason to make the text container width have a limit? I feel like it looks weird not having it fill the space. Aside from that, everything seems good :100:

Reply:
@KateAThornley made a mockup that seemed to have the max width limited to around 525px. Maybe she can respond and say if there was reason for that or not. The widest version of it shown in the above image is on a 1440p monitor, bigger than most govt employees would be using, so that makes it look a fair bit weirder

Reply:
ah okok, I just tried it locally and was wondering why it was not the full width

Reply:
O noo in my mockup lorem ipsum had an enter space in it so it was a newline not a max width shoot my bad I shiouldve specified that